content/en/docs/apps/port-broker.md

```diff
@@ -16,6 +16,8 @@ configuring the port mappings of your container.
 
 If `proxy` is specified and set to `true`, this will open a tunnel that will forward all traffic incoming onto the port exposed by the container.
 
+If your outbound data using Port Broker requires a specific  protocol, add `scheme` followed by the desired protocol.
```

Comment:
Remove the extra blank space after `specific`.

Reply:
Done!